test/ml/test_ml.py

```diff
@@ -25,6 +26,14 @@ def test_summarize_networks(self):
         dataframe.to_csv(OUT_DIR + 'dataframe.csv')
         assert filecmp.cmp(OUT_DIR + 'dataframe.csv', EXPECT_DIR + 'expected-dataframe.csv', shallow=False)
 
+    def test_summarize_networks_empty(self):
+        with pytest.raises(ValueError): #raises error if empty dataframe is used for post processing
+            ml.summarize_networks([INPUT_DIR + 'test-data-empty/empty.txt'])
```

Comment:
Should there be an empty file added with this name? I don't see it in the list of files to add in the pull request. Currently I expect this to return an OSError instead of the new ValueError.

Reply:
I believe this test isn't running. It has the same name as the next test, so the function is redefined.

Reply:
Now I see that https://github.com/Reed-CompBio/spras/blob/master/test/ml/input/test-data-empty/empty.txt is already in the repo